tests/acceptance/features/coreApiTrashbinRestore/trashbinRestore.feature

```diff
@@ -35,7 +35,7 @@ Feature: Restore deleted files/folders
       | new      |
 
 
-  Scenario Outline: A file deleted from a folder can be restored to the original folder
+  Scenario Outline: a file deleted from a folder can be restored to the original folder
```

Comment:
@PrajwolAmatya remove articles from all the scenario titles, there are a lot in this feature file
```suggestion
  Scenario Outline: file deleted from a folder can be restored to the original folder
```